### PR TITLE
Fixes for Product update 31102022

### PR DESCRIPTION
**Pull request by MartaD**

### CODE REVIEW

File: docs/change-log.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-![advanced_image_management](_assets/images/2022-10-31_Advanced_Image_Management.png)
+- **New:** Introduced a new API and updated UI which allow for image management at scale. Users can now identify, filter, archive and update images and their versions with ease. More detailed information on new capabilities can be found in the [feature announcement article](https://humanitec.com/blog/container-image-management-made-hassle-free-feature-announcement) on Humanitec's blog. <br> <img width="480" alt="2022-10-31_Advanced_Image_Management" src="./_assets/images/2022-10-31_Advanced_Image_Management.png">
 
-- **Fixed:** Provisioning a resource with an invalid [placeholder](https://docs.humanitec.com/reference/concepts/app-config/placeholders) now returns an expected error code with more descriptive error message.  
+
+- **Fixed:** Provisioning a resource with an invalid [placeholder](https://docs.humanitec.com/reference/concepts/app-config/placeholders) now returns the expected error code with a more descriptive message.
```
Comment (MartaD) [inline docs/change-log.md:10]: @sujaya-sys Let me know what you decide on this comment and I'll update accordingly:

> I'd reword this to:
> 
> "now returns an error code with a descriptive error message."
> 
> Also "improved" might make more sense here as a category (compared to "fixed")

My response: the issue was that it returned an incorrect error code, 500 instead of 400, and that was fixed. Happy to change to your suggestion if it's not that important or clear from my text.

Comment (sujaya-sys) [reply docs/change-log.md:10]: makes sense! I'll resolve my comment :-)

**Review by sujaya-sys — APPROVED** [review]
LGTM
